tests/cupyx_tests/scipy_tests/sparse_tests/test_linalg.py

```diff
@@ -451,12 +452,6 @@ def _test_gmres(self, dtype, xp, sp, a, M):
             return sp.linalg.gmres(
                 a, b, x0=x0, restart=self.restart, M=M, atol=atol)
 
-    @testing.for_dtypes('fdFD')
-    @testing.numpy_cupy_allclose(rtol=1e-5, atol=1e-5, sp_name='sp')
-    def test_dense(self, dtype, xp, sp):
-        a, M = self._make_matrix(dtype, xp)
-        return self._test_gmres(dtype, xp, sp, a, M)
-
```

Comment:
It looks like a test for `gmres` is being mistakenly removed. Could you revert it?

Reply:
sorry about that @anaruse , will rectify right away